### ISSUE TITLE
Upgrade to Prophet 1.1

### ISSUE THREAD

**Issue by tcuongd** [facebookresearch/Kats#249]
Hello! We recently released v1.1 of Prophet: https://github.com/facebook/prophet/releases/tag/v1.1 which removes the `pystan==2.19.1.1` dependency and has binary distributions (python 3.7-3.10) uploaded to PyPI. I think specifying `prophet >= 1.1` will make the installation process for Kats a lot smoother.

**Comment by noreplacementfordisplacement** [comment]
> Hello! We recently released v1.1 of Prophet: https://github.com/facebook/prophet/releases/tag/v1.1 which removes the `pystan==2.19.1.1` dependency and has binary distributions (python 3.7-3.10) uploaded to PyPI. I think specifying `prophet >= 1.1` will make the installation process for Kats a lot smoother.

How do I do that? pip install kats prophet >= 1.1

**Comment by octavd** [comment]
Hello!
Any news regarding this? Looking forward.

**Comment by inigohidalgo** [comment]
> > Hello! We recently released v1.1 of Prophet: https://github.com/facebook/prophet/releases/tag/v1.1 which removes the `pystan==2.19.1.1` dependency and has binary distributions (python 3.7-3.10) uploaded to PyPI. I think specifying `prophet >= 1.1` will make the installation process for Kats a lot smoother.
> 
> How do I do that? `pip install kats prophet >= 1.1` ?

It isn't easy because the package has changed names from `fbprophet` to `prophet`. What I've done temporarily until this is updated is to download the source code for kats and edit it to match this new package name myself like so:

https://github.com/facebookresearch/Kats/pull/252/files

And `pip install project/directory/`

**Comment by Vishnu-Rangiah** [comment]
Please upgrade to prophet 1.1

**Comment by michaelbrundage** [comment]
We're currently blocked on performing this upgrade due to some internal incompatibilities. Looking forward to upgrading to Prophet 1.1 as soon as those issues are worked through.

**Comment by michaelbrundage** [comment]
Duplicate of #62

**Comment by octavd** [comment]
Hello @michaelbrundage 
Any news regarding the upgrade to Prophet 1.1 for kats?
